docs/FEATURES.md

```diff
@@ -1,32 +1,75 @@
-| Frequency-based Services | Frequencies               | One line of data in [frequencies.txt](https://gtfs.org/schedule/reference/#frequenciestxt)                                                                                                                                                                                                                                                                                                                                                        |
+| Feature Group              | Feature                   | How is the presence of a feature determined (minimum requirements)  |   Documentation Link                                                                                                                                                                                                                                                                                                                                                                            |
+|--------------------------|---------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---|
+| Accessibility             | Text-to-speech            | One **tts_stop_name** value in [stops.txt](https://gtfs.org/schedule/reference/#stopstxt)  |   https://gtfs.org/getting_started/features/accessibility/#text-to-speech |
+| Accessibility             | Stops Wheelchair Accessibility   | One **wheelchair_boarding** value in [stops.txt](https://gtfs.org/documentation/schedule/reference/#stopstxt) |  https://gtfs.org/getting_started/features/accessibility/#stops-wheelchair-accessibility |
+| Accessibility             | Trips Wheelchair Accessibility   | One **wheelchair_accessible** value in [trips.txt](https://gtfs.org/documentation/schedule/reference/#tripstxt) |   https://gtfs.org/getting_started/features/accessibility/#trips-wheelchair-accessibility |
+| Base add-ons            | Route Colors              | One **color** value <br>OR<br>one text_color value in [routes.txt](https://gtfs.org/schedule/reference/#routestxt)    |    https://gtfs.org/getting_started/features/base_add-ons/#route-colors |
```

Comment:
Is it possible that `color` has been changed to `route_color` and `text_color` changed to `route_text_color` in routex.txt?

Reply:
On the spec side I believe they have always been named as `route_color` and `route_text_color`